libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -54,6 +55,9 @@ struct abi_serializer {
    type_name get_action_type(name action)const;
    type_name get_table_type(name action)const;
 
+   optional<map<uint64_t, string>::const_iterator>  find_error_message( uint64_t error_code )const;
+   const string&                                    get_error_message( uint64_t error_code )const;
```

Comment:
I think returning a string would be safer. The abi_serializer objects can be short-lived.